decidim-core/app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -2,4 +2,4 @@
 
 <p class="email-instructions"><%= t(".instruction") %></p>
 
-<p class="email-button email-button__cta"><%= link_to t(".action"), confirmation_url(@resource, confirmation_token: @token, host: @resource.organization.host) %></p>
+<p class="email-button email-button__cta"><%= link_to t(".action"), confirmation_url(@resource, confirmation_token: @token, host: @resource.organization.host, protocol: (Rails.env=="production" and Decidim.force_ssl)?"https":"http") %></p>
```

Comment:
We should move the protocol definition in an helper like : 
```suggestion
confirmation_url(@resource, confirmation_token: @token, host: @resource.organization.host, protocol: protocol) 
```

Maybe we could reuse the method defined in `decidim-core/app/mailers/concerns/decidim/multitenant_asset_host.rb` ?